torchdynamo/variables/torch.py

```diff
@@ -76,6 +76,8 @@ def as_python_constant(self):
     def can_constant_fold_through(self):
         if self.value in (
             torch.device,
+            torch.finfo,
+            torch.iinfo,
```

Comment:
Neat, I have a branch where I did this manually, (voz/huggingface_attr_hax) this is much better.